src/pages/upload/index.tsx

```diff
@@ -191,7 +199,9 @@ const Upload: React.FC<
   const isIncomplete =
     noAttachedFiles || anyFilesCurrentlyUploading || values.course.title === ''
 
-  const submitDisabled = isIncomplete || isSubmitting
+  const submitWasSuccessful = submitResponse?.status === 200
+
+  const submitDisabled = isIncomplete || isSubmitting || submitWasSuccessful
```

Comment:
Disable the submit button if the form submitted successfully, don't want to allow a double-submit after the fact.